### DISCUSSION TITLE
Template doesn't use site images

### DISCUSSION THREAD

**Discussion by igasparetto** [bootscore/bootscore#300] [Q&A]
**Favicon**

I had to add the following line to the child template. I think it should be fixed in the main template as well.
`<?php wp_site_icon(); ?>`

**Site Logo**
The option is not even appearing in the Customize > Site identity.
I don't know why yet.

**Comment by crftwrk** [comment]
Theme doesn't use icon in customizer, because customizer uses one image for all devices. But you need different images for favicon or touch icons. That's why those icons are linked in the `header.php` to make sure that all devices have their correct icons. Check here https://bootscore.me/documentation/header-php/#Favicon_touch-icons_and_theme_colors

Use https://realfavicongenerator.net to generate the icons and replace them in your child folder `img/favicon`.

Does this help?

**Reply by igasparetto** [reply]
I did read that but I don't think that what is proposed is a good solution.
People want to be able to edit their favicon and logo using the backend Customize interface.

**Reply by igasparetto** [reply]
I think this is an Issue, not a Discussion.

**Reply by crftwrk** [reply]
Favicons are handled the same way as the svg logo. bootScore is a developer theme. 

> I think this is an Issue, not a Discussion.

Please open a new issue again, then we can discuss there and check what other users think about this. Can't convert this discussion back to an issue.
